algonaut_client/src/algod/v1/mod.rs

```diff
@@ -14,14 +16,22 @@ const AUTH_HEADER: &str = "X-Algo-API-Token";
 
 /// Client for interacting with the Algorand protocol daemon.
 pub struct Client {
-    pub(super) url: String,
-    pub(super) token: String,
-    pub(super) headers: HeaderMap,
-    pub(super) http_client: reqwest::Client,
+    url: String,
+    token: String,
+    headers: HeaderMap,
+    http_client: reqwest::Client,
 }
 
 impl Client {
-    /// Returns Ok if healthy
```

Comment:
I removed them because they're now on the user interfacing api ([e.g.](https://github.com/manuelmauro/algonaut/pull/41/files#diff-56a5d3cc7625b762de41f501a93206c306678faaebca41a7422b83404890a144R30)) and maintaining duplicate documentation is error prone. If we do major changes to this api in the future, we can reconsider maintaining separate documentation for the client, but for now this seems fine.

Reply:
Another thing I thought was that since the client is now crate private, it's only for us, and we can look in the developer portal, which also might be more up to date.

Reply:
Ok, got the reasoning now. I am fine removing them now even if this way the crate `algonaut_client` loses any docs concerning those functions. We may come up with some different approach in the future.